README.md

```diff
@@ -8,7 +8,24 @@ MIOpen supports two programming models -
 1. [HIP](https://github.com/ROCm-Developer-Tools/HIP) (Primary Support).
 2. OpenCL.
 
+## Documentation
+
+For a detailed description of the **MIOpen** library see the [Documentation](https://rocmdocs.amd.com/projects/MIOpen/en/latest/).
+
+### How to build documentation
+
+Run the steps below to build documentation locally.
+
+```
```

Comment:
If docs can build with cmake, we should include that as an alternate instruction.